common/management/src/namespace/namespace_mgr.rs

```diff
@@ -32,109 +34,208 @@ pub static NAMESPACE_API_KEY_PREFIX: &str = "__fd_namespaces";
 #[allow(dead_code)]
 pub struct NamespaceMgr {
     kv_api: Arc<dyn KVApi>,
+    lift_time: Duration,
+    namespace_prefix: String,
 }
 
 impl NamespaceMgr {
-    #[allow(dead_code)]
-    pub fn new(kv_api: Arc<dyn KVApi>) -> Self {
-        NamespaceMgr { kv_api }
+    pub fn new(
+        kv_api: Arc<dyn KVApi>,
+        tenant: &str,
+        namespace: &str,
+        lift_time: Duration,
+    ) -> Result<Self> {
+        Ok(NamespaceMgr {
+            kv_api,
+            lift_time,
+            namespace_prefix: format!(
+                "{}/{}/{}/databend_query",
```

Comment:
For the prefix: `"{}/{}/{}/databend_query"`,  the suffix `databend_query` is used for?